### DISCUSSION TITLE
weldx file suffix

### DISCUSSION THREAD

**Discussion by CagtayFabry** [BAMWelDX/weldx#404] [General]
Using the `WeldxFile` class it is no longer required to use the `asdf` library or really be familiar with the `asdf` API

To make things easier for the user, I suggest we use a distinct file suffix for `weldx` files, such as `.weldx` or `.wx` (or both)

What do you think @marscher @vhirtham ?

**Comment by vhirtham** [comment]
It's about time, I guess 😉 Would support both file extensions

**Reply by marscher** [reply]
What do you mean by "support"? Its really a users choice how he or she names files. I guess you already started to recommend the ".weldx" extension in our tutorials. Since we are not bound to 8.3 naming anymore I'd say stick to .weldx

**Reply by vhirtham** [reply]
I meant supporting `.weldx` and `.wx` for weldx files, like you can write `.yaml` or `.yml`. I just referred to: 

> I suggest we use a distinct file suffix for weldx files, such as .weldx or .wx **(or both)**

Yes, the user can name the file as he likes, but I think the question is what we want to propose as a file extension in our publications and resources.
